### PR TITLE
Curve symbol and LLama price

### PR DESCRIPTION
**Pull request by 0xtekgrinder**
- Update the system to get the curve symbol from plain etherjs to subgraph
- Change the price api used from coingecko to defi llama to get more accurate data + it will not be affected if coingecko is down. Also added coingecko as callback

### CODE REVIEW

**Comment by startonmateo** [comment]
Decided to go with the curve api directly instead of a subgraph as it's much cleaner

**Comment by 0xtekgrinder** [comment]
Decided to go with the curve api directly instead of a subgraph as it's much cleaner

File: src/scripts/getSymbolFromGauge.ts [hunk 1]
```diff
@@ -89,43 +91,43 @@ const getRecipient = async (gauge: string) => {
...
-    const gaugeContract = new Contract(gauge, CurveGaugeAbi, etherProvider);
-    const lpAddress = await gaugeContract.lp_token();
+    const res = await axios.get('https://api.curve.fi/api/getAllGauges');
 
-    const lpContract = new Contract(lpAddress, ERC20, etherProvider);
-    symbol = await lpContract.symbol();
-  } catch (e) {
-    console.error(e);
+    for (const gauge in res.data.data) {
+      if (res.data.data[gauge].gauge == expectedGauge) {
```
[the remark this answers is not in the document]
Comment (0xtekgrinder) [reply src/scripts/getSymbolFromGauge.ts]: Fixed the double quotes but what do you mean by formatted ? Is it the split and if it's not formatted how is it ?

[the remark this answers is not in the document]
Comment (0xtekgrinder) [reply src/scripts/getSymbolFromGauge.ts]: Oh ok got it making the fix now 👍

**Pull request merged**